docs/workflows/partnership-pages.md

```diff
@@ -2,21 +2,28 @@
 
 ## NU (WES):
 
-- Create a new page
-- Add content from copy
+- Go to [NU Pre-production Site](https://nu-edu-preprod.go-vip.co/wp-admin/)
+- [Add or edit a partnership page](https://nu-edu-preprod.go-vip.co/wp-admin/edit.php?post_type=partnership)
+- [Compress any given PDFs](https://smallpdf.com/)
+- [Compress any given images and logos if necessary](https://compressor.io/)
+- Beautify the name of the files. Example: `NU_Tulare_County_Flyer_06.2021.pdf` to `Tulare_County_Flyer.pdf`
+- Add the content given by copy
 - Update `Page Attributes`:
-    - Parent: `Workfoce Education Solutions`
+    - Parent: `No Parent`
     - Template: `Standard Hero Sidebar`
+    - Order: `0`
     - Request `Campaign Activity` and `Organization` field values:
         - Once page is in development, add the URL in the [NU_Partnerships_Organizational](https://docs.google.com/spreadsheets/d/1CEYuLWvMCTCFmRUPbL8ZS9cw-igYbr7VZauYH1utfhQ/edit?ts=5e7e76d5#gid=1054635506) and notify April Resurreccion (to direct a comment type `+` followed by her email address: AResurreccion@nu.edu)
-        - April will add the **Organization MKT_WES** and **Campaign Activity** values in the spreadsheet and notify web dev.
+        - April will add the **Organization MKT_WES** and **Campaign Activity** values in the spreadsheet and notify web dev
         - Apply these values to the `Form Setup` metabox
             - **Campaign Activity** field value
             - **Organization** field value
-        - When page is ready for testing, include April as an approver to the project (2nd to last approver before Jordan) to ensure values are being populated correctly and that data is passed through from Eloqua to CRM
-- There is no need to add a hero image, it should be automatically displayed. Unless otherwise noted in the job description.
-- If you need to add the partner logo to the sidebar, create a `custom field` with the name `sidebar_content` and add any HTML you need in the value field.  
-(i.e. `<img src="/wp-content/uploads/xxxx/xx/partner_logo.png" class="any-class-here" alt="The Partner logo.">` - **Do not forget to include the `alt` attribute.**)
+    - There is no need to add a hero image. It should be automatically displayed, unless otherwise noted in the job description

```

Comment:
I know this already existed but I think the following will be clearer:
```suggestion
    - Unless otherwise noted in the job description, there is no need to add a hero image. It should be automatically displayed from the "parent" `/partnerships/`.
```